### ISSUE TITLE
Some Field names are missing leading Character

### ISSUE THREAD

**Issue by degill** [cmelchior/realmfieldnameshelper#6]
At the moment I dont know any further information than this:

For some of my fields, the created static field is missing a leading Character.

Example: 

```
  public static final String ID_QUEST_PACK = "idQuestPack";
  public static final String TEMID_ITEM = "itemIdItem";
  public static final String TEMID_PRE_CONDITION = "itemIdPreCondition";
  public static final String TEMMODE = "itemMode";
```

It seems that this is always happening for fields beginning with "item"

**Comment by cmelchior** [comment]
Can your post your Realm model class as well?

**Comment by degill** [comment]
Minimal example:

```
public class Test extends RealmObject {
    private int itemQuantityDelta;
}
```

This behaviour is not happening if the field is just called `item`

**Comment by cmelchior** [comment]
Thanks. I will take a look

**Comment by cmelchior** [comment]
Fixed by #7
